.github/workflows/tests_gpu_python.yml

```diff
@@ -197,9 +197,18 @@ jobs:
         if: matrix.pl_backend == 'lightning_gpu'
         run: |
           DEVICENAME=`echo ${{ matrix.pl_backend }} | sed "s/_/./g"`
-          pl-device-test --device ${DEVICENAME} --skip-ops --shots=20000 $COVERAGE_FLAGS --cov-append
+          pl-device-test --device ${DEVICENAME} --skip-ops --shots=10000 $COVERAGE_FLAGS --cov-append
```

Comment:
Do we need to double-check the flakiness of these integration tests on all devices?

Reply:
I don't know what you mean with that. Also I didn't add any change to how we test.

Reply:
Solved off-line. I'm testing the flakness of the tests for the reduced number of shots.